src/Error/Renderer/Page.php

```diff
@@ -105,17 +106,13 @@ public function render(Exception $error)
 
 			$this->document->setTitle(\Lang::txt('Error') . ': ' . $error->getCode());
 
-			$path = PATH_APP . DS . 'templates';
-			if (!is_dir($path . DS . $this->template))
-			{
-				$path = PATH_CORE . DS . 'templates';
-			}
+			$template = $this->template->load();
```

Comment:
This is my only complaint...$this->template defaults to null, so this could result in an error.